analyses/new-feature-template.qmd

```diff
@@ -0,0 +1,1564 @@
+
+### `r township_names[3]`
+
+
+```{r, results = 'asis'}
+shap_tables[[3]]
+```
+
+### `r township_names[4]`
+
+
+```{r, results = 'asis'}
+shap_tables[[4]]
+```
+### `r township_names[5]`
+
+
+```{r, results = 'asis'}
+shap_tables[[5]]
+```
+
+### `r township_names[6]`
+
+```{r, results = 'asis'}
+shap_tables[[6]]
+```
+
+###  `r township_names[7]`
+
+```{r, results = 'asis'}
+shap_tables[[7]]
+```
+
+###  `r township_names[8]`
+
+```{r, results = 'asis'}
+shap_tables[[8]]
+```
+
+###  `r township_names[9]`
+
+```{r, results = 'asis'}
+shap_tables[[9]]
+```
+
+###  `r township_names[10]`
+
+```{r, results = 'asis'}
+shap_tables[[10]]
+```
+
+###  `r township_names[11]`
+
+```{r, results = 'asis'}
+shap_tables[[11]]
+```
+###  `r township_names[12]`
+
+```{r, results = 'asis'}
+shap_tables[[12]]
+```
+
+###  `r township_names[13]`
+
+```{r, results = 'asis'}
+shap_tables[[13]]
+```
+
+###  `r township_names[14]`
+
+```{r, results = 'asis'}
+shap_tables[[14]]
+```
+
+###  `r township_names[15]`
+
+```{r, results = 'asis'}
+shap_tables[[15]]
+```
+
+###  `r township_names[16]`
+
+```{r, results = 'asis'}
+shap_tables[[16]]
+```
+
+###  `r township_names[17]`
+
+```{r, results = 'asis'}
+shap_tables[[17]]
+```
+
+###  `r township_names[18]`
+
+```{r, results = 'asis'}
+shap_tables[[18]]
+```
+
+###  `r township_names[19]`
+
+```{r, results = 'asis'}
+shap_tables[[19]]
+```
+
+###  `r township_names[20]`
+
+
+```{r, results = 'asis'}
+shap_tables[[20]]
+```
+
+###  `r township_names[21]`
+
+```{r, results = 'asis'}
+shap_tables[[21]]
+```
+###  `r township_names[22]`
+
+```{r, results = 'asis'}
+shap_tables[[22]]
+```
+
+###  `r township_names[23]`
+
+```{r, results = 'asis'}
+shap_tables[[23]]
+```
+
+###  `r township_names[24]`
+
+```{r, results = 'asis'}
+shap_tables[[24]]
+```
+
+###  `r township_names[25]`
+
+```{r, results = 'asis'}
+shap_tables[[25]]
+```
+
+###  `r township_names[26]`
+
+```{r, results = 'asis'}
+shap_tables[[26]]
+```
+
+###  `r township_names[27]`
+
+```{r, results = 'asis'}
+shap_tables[[27]]
+```
+
+###  `r township_names[28]`
+
+```{r, results = 'asis'}
+shap_tables[[28]]
+```
+
+###  `r township_names[29]`
+
+```{r, results = 'asis'}
+shap_tables[[29]]
+```
+
+###  `r township_names[30]`
+
+```{r, results = 'asis'}
+shap_tables[[30]]
+```
+
+###  `r township_names[31]`
+
+```{r, results = 'asis'}
+shap_tables[[31]]
+```
+###  `r township_names[32]`
+
+```{r, results = 'asis'}
+shap_tables[[32]]
+```
+
+###  `r township_names[33]`
+
+```{r, results = 'asis'}
+shap_tables[[33]]
+```
+
+###  `r township_names[34]`
+
+```{r, results = 'asis'}
+shap_tables[[34]]
+```
+
+###  `r township_names[35]`
+
+```{r, results = 'asis'}
+shap_tables[[35]]
+```
+
+###  `r township_names[36]`
+
+```{r, results = 'asis'}
+shap_tables[[36]]
+```
+
+###  `r township_names[37]`
+
+```{r, results = 'asis'}
+shap_tables[[37]]
+```
+
+###  `r township_names[38]`
+
+```{r, results = 'asis'}
+shap_tables[[38]]
+```
+
+:::
+
+## SHAP Comparison
```

Comment:
**suggestion:** I would scrap this since it didn't yield anything useful, though its utility may improve if you don't enable CV and use the same hyperparams across the baseline v new run.

Reply:
I agree, I left everything in for the time being in case you had thoughts on it.